docs/core/versions/lts-current.md

```diff
@@ -0,0 +1,34 @@
+---
+title: .NET Core Support
+description: LTS and Current Release Train Support
+keywords: .NET, .NET Core, lts, current, fts, support, support trains, support tracks, Lifecycle, release trains
+author: kendrahavens
+ms.date: 01/30/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: fedc7025-f320-4cba-957b-ef74885f66de
+---
+
+# .NET Core Support
+
+This is a general description of .NET Core support. For more detail see the original [.NET Blog post introducing the lts and current tracks with the 1.0 release.](https://blogs.msdn.microsoft.com/dotnet/2016/07/26/net-support-and-versioning/)
+
+## LTS and Current Release Trains
+Long Term Support (LTS) and Current are concepts in use throughout the software world and should be familiar to many. LTS releases are maintained for stability over their lifecycle, receiving fixes for important issues and security fixes. New feature work and additional bug fixes take place in Current releases. From a support perspective, these release trains have the following support lifecycle attributes.
+
+LTS releases are
+* Supported for three years after the general availability date of a LTS release
+* And one year after the general availability of a subsequent LTS release
```

Comment:
Good catch! It changed on our Lifecycle fact sheet but not in the original blog post!